packages/config/src/api/site_info.ts

```diff
@@ -126,14 +129,16 @@ const getIntegrations = async function ({
   offline,
   token,
   featureFlags,
+  netlifyApiHost,
 }: GetIntegrationsOpts): Promise<IntegrationResponse[]> {
   if (!siteId || offline) {
     return []
   }
   const sendBuildBotTokenToJigsaw = featureFlags?.send_build_bot_token_to_jigsaw
   const { host } = testOpts
-
-  const baseUrl = new URL(host ? `http://${host}` : `https://api.netlifysdk.com`)
+  const extensionApiBaseUrl =
+    netlifyApiHost === 'api.netlify.com' ? 'https://api.netlifysdk.com' : `https://api-staging.netlifysdk.com `
```

Comment:
could we extract these constants somewhere instead of duplicating them in two files? presumably they could just be exported from `src/api/integrations.ts`

Reply:
I've moved them to a top level for now